main.go

```diff
@@ -81,14 +117,19 @@ func main() {
 
 	http.HandleFunc("/", handler)
 
-	log.Printf("http serving at %s", urlAddr)
-
-	if !*skipBrowser {
-		go openBrowser(urlAddr)
-	}
-
-	if err := http.ListenAndServe(httpAddr, nil); err != nil {
-		log.Fatal(err)
+	switch *outputFormat {
+	case "svg":
+		outputDot(*outputFile, *outputFormat)
+	case "png":
+		outputDot(*outputFile, *outputFormat)
+	default:
```

Comment:
You realize that this will fallback to running http server when anything else beside `svg` and `png` is chosen without any warning for the user even though the documentation states that omitting the `format` flag runs the server mode. I think this switch should be enclosed by a condition that `outputFormat` is not empty.